activiti-cloud-services-audit-jpa/src/main/java/org/activiti/cloud/services/audit/jpa/conf/AuditJPAAutoConfiguration.java

```diff
@@ -269,20 +270,7 @@ public TimerRetriesDecrementedEventConverter timerRetriesDecrementedEventConvert
     
     @ConditionalOnMissingBean
     @Bean
-    public MessageReceivedEventConverter messageReceivedEventConverter(EventContextInfoAppender eventContextInfoAppender) {
```

Comment:
why do not we need the converters related to messages anymore?

Reply:
Thank you for this tip (wrongly removed)... Fixed now!